src/PageLayout/PageLayout.tsx

```diff
@@ -362,6 +362,7 @@ export type PageLayoutPaneProps = {
    */
   dividerWhenNarrow?: 'inherit' | 'none' | 'line' | 'filled'
   sticky?: boolean
+  stickyTop?: number // the height of the sticky top element
```

Comment:
As a start, we only make the type number and thought about adding string as well for the values that includes units. Thoughts?